crates/rslint_core/src/groups/errors/no_duplicate_imports.rs

```diff
@@ -0,0 +1,133 @@
+use std::collections::HashMap;
+
+use crate::rule_prelude::*;
+use ast::{ExportDecl, ImportDecl, Literal, Source};
+
+declare_lint! {
+    /**
+    Disallow duplicate imports.
+
+    Multiple import statements with the same source can be combined to one statement. This improves readability.
+
+    ## Incorrect Code Examples
+
+    ```js
+    import { foo } from "bla";
+    import { bar } from "bla";
+    ```
+
+    ## Including Exports
+
+    ### Incorrect Code Examples
+
+    ```js
+    import { foo } from "bla";
+    export { foo } from "bla";
+    ```
```

Comment:
Can you squash all of these codeblocks into a single one and write a comment "including exports"